setup.py

```diff
@@ -516,7 +524,7 @@ def detect_modules(self):
             add_dir_to_list(self.compiler.library_dirs, '/usr/local/lib')
             add_dir_to_list(self.compiler.include_dirs, '/usr/local/include')
         # only change this for cross builds for 3.3, issues on Mageia
-        if cross_compiling:
+        if ( cross_compiling and not host_platform == 'vxworks'):
```

Comment:
The extra space after the paren here is probably the source of the patchcheck warning on TravisCI.